### DISCUSSION TITLE
[FEATURE]建议将GPU监控集成到概览和主机监控中

### DISCUSSION THREAD

**Discussion by sagezhj** [1Panel-dev/1Panel#6023] [Ideas]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

专业版虽然有了GPU监控，但是目前功能较弱，基本就是nvidia-smi命令的页面展示。希望后续可以将相关信息集成进概览和主机监控中，可以在概览中直观看到当前GPU使用率，特别是可以在主机监控中看到GPU的历史使用情况非常有用。

### 请描述你建议的实现方案

在GPU页面中设置监控间隔，如1秒或者5秒。如果系统开销很大就增加监控开关，或者设置较大默认值。然后就可以在概览和监控中看到GPU的使用情况

**Comment by wanghe-fit2cloud** [comment]
在概览页面中有 GPU 相关信息，如图：
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/ec01ec60-6a12-4737-b445-63961aeb5560">
